src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -50,14 +52,20 @@ def _check_conditional_args(
             parser.error(
                 "The --endpoint option is required when using the 'openai' service-kind."
             )
-        if args.endpoint == "v1/chat/completions":
-            args.output_format = OutputFormat.OPENAI_CHAT_COMPLETIONS
-        elif args.endpoint == "v1/completions":
-            args.output_format = OutputFormat.OPENAI_COMPLETIONS
+        else:
+            if args.endpoint == "chat":
+                args.output_format = OutputFormat.OPENAI_CHAT_COMPLETIONS
+            elif args.endpoint == "completions":
+                args.output_format = OutputFormat.OPENAI_COMPLETIONS
+            if args.port:
+                args.endpoint = args.port.lstrip(" /")
```

Comment:
should we strip on the right side as well?

Reply:
i was hoping to deal with the possibility that a user adds a beginning / and then we add another in PA, which then breaks things.
I am not as eager to mess with the right side, since i dont know every usage case.